### DISCUSSION TITLE
Stock Quantity going in Minus

### DISCUSSION THREAD

**Discussion by maazbinrizwan** [opensourcepos/opensourcepos#4051] [Q&A]
In the sales view where  we scan any item and it is added to the card. When the stock is 0 it still adds it to the cart and from the inventory the item goes from 0 to -1. I want the functionality that when the item is 0 it cant be added to the cart with the message that stock is empty or something. I need this to implement in a project. Please help i have timeline.

![Screenshot 2024-08-30 114101](https://github.com/user-attachments/assets/7aac9f68-e7b5-4b70-aa92-591e78da5c84)

**Comment by jekkos** [comment]
There might be an option already to enforce this check config section

**Reply by maazbinrizwan** [reply]
> There might be an option already to enforce this check config section

Hey thanks for replying. Can you please send me a screen shot of where can i find it please. Would much appreciate. I cant find the option
